src/widget/wtrackmenu.h

```diff
@@ -10,6 +10,7 @@
 #include "library/trackprocessing.h"
 #include "preferences/usersettings.h"
 #include "track/beats.h"
+#include "track/track.h"
```

Comment:
instead of including the entire header here, can Track be forward-declared?

Reply:
Yes. I have deleted this line.